l10n_br_account/res_partner.py

```diff
@@ -17,43 +17,40 @@
 #along with this program.  If not, see <http://www.gnu.org/licenses/>.        #
 ###############################################################################
 
-from openerp.osv import orm, fields
-
-FISCAL_POSITION_COLUMNS = {
-    'name': fields.char('Fiscal Position', size=128, required=True),
-    'fiscal_category_id': fields.many2one(
-        'l10n_br_account.fiscal.category', 'Categoria Fiscal'),
-    'fiscal_category_fiscal_type': fields.related(
-        'fiscal_category_id', 'fiscal_type', type='char', readonly=True,
-        relation='l10n_br_account.fiscal.category', store=True,
-        string='Fiscal Type'),
-    'type': fields.selection([('input', 'Entrada'), ('output', 'Saida')],
-                             'Tipo'),
-    'type_tax_use': fields.selection(
-        [('sale', 'Sale'), ('purchase', 'Purchase'), ('all', 'All')],
-        'Tax Application'),
-    'inv_copy_note': fields.boolean(u'Copiar Observação na Nota Fiscal'),
-    'asset_operation': fields.boolean(u'Operação de Aquisição de Ativo',
-        help=u"""Caso seja marcada essa opção, será incluido o IPI na base de
-            calculo do ICMS."""),
-    'state': fields.selection([('draft', u'Rascunho'),
-            ('review', u'Revisão'), ('approved', u'Aprovada'),
-            ('unapproved', u'Não Aprovada')], 'Status', readonly=True,
-            track_visibility='onchange', select=True),
-}
+from openerp import api, models, fields
 
-FISCAL_POSITION_DEFAULTS = {
-    'state': 'draft',
-}
 
-
-class AccountFiscalPositionTemplate(orm.Model):
+class AccountFiscalPositionTemplate(models.Model):
     _inherit = 'account.fiscal.position.template'
-    _columns = FISCAL_POSITION_COLUMNS
-    _defaults = FISCAL_POSITION_DEFAULTS
+
+    name = fields.Char(string='Fiscal Position', size=128, required=True)
+    fiscal_category_id = fields.Many2one(
+        'l10n_br_account.fiscal.category', string='Categoria Fiscal')
+    fiscal_category_fiscal_type = fields.Selection(
+        related='fiscal_category_id.fiscal_type', readonly=True,
+        store=True,
+        string='Fiscal Type')
+    type = fields.Selection([('input', 'Entrada'), ('output', 'Saida')],
+                             string='Tipo')
+    type_tax_use = fields.Selection(
+        [('sale', 'Sale'), ('purchase', 'Purchase'), ('all', 'All')],
+        string='Tax Application')
+    inv_copy_note = fields.Boolean(string=u'Copiar Observação na Nota Fiscal')
+    asset_operation = fields.Boolean(string=u'Operação de Aquisição de Ativo',
+        help=u"""Caso seja marcada essa opção, será incluido o IPI na base de
+            calculo do ICMS.""")
+    id_dest = fields.Selection([('1', u'Operação interna'),
```

Comment:
This filed id_dest should be in l10n_br_account.product.cfop object, I did this in https://github.com/akretion/l10n-brazil/blob/migration/l10n_br_account_product/l10n_br_account_product/models/l10n_br_account_product.py#L53